.pipelines/OneBranch.Official.yml

```diff
@@ -83,9 +83,16 @@ extends:
             set PYTHONPATH=
             set PYTHONHOME=
             python -m pip install --upgrade pip
-            python -m pip install cibuildwheel
+            python -m pip install cibuildwheel numpy
             python -m cibuildwheel --platform windows --output-dir $(REPOROOT)\out
           displayName: Build wheels
+        - task: SDLNativeRules@3
+          inputs:
+            msBuildArchitecture: amd64
+            setupCommandlines: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.EXE" $(REPOROOT) -A x64 -B $(REPOROOT)\windows_out -DOCOS_ENABLE_PYTHON=ON -DOCOS_ENABLE_CTEST=OFF -DCMAKE_BUILD_TYPE=Release'
+            msBuildCommandline: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\amd64\msbuild.exe" "$(REPOROOT)\windows_out\onnxruntime_extensions.sln" /p:RunCodeAnalysis=true /p:platform=x64 /p:configuration=Release /p:VisualStudioVersion="16.0" /m /p:PreferredToolArchitecture=x64'
+            excludedPaths: '$(Build.BinariesDirectory)#$(Build.SourcesDirectory)\cmake#C:\program files (x86)'
+          displayName: 'Run the PREfast SDL Native Rules for MSBuild'
         - script: |
             7z x $(REPOROOT)\out\*.whl -o$(REPOROOT)\out\*
           displayName: unzip the package
```

Comment:
You might need to append a "Guardian Break" task to detect if PREfast outputs any warnings.

Reply:
There are already hundred of warnings caused by external 3rd party libraries. Won't expect it will be fixed.
Does PREfast only output warnings only, or something severer will be outputted as error?

Reply:
_deps are in your build folder. See my comments above. Yes, we should exclude them.

Reply:
This task output warnings only.  If you run it locally without this ADO task, warnings are errors

Reply:
Guardian Break task was added.

Reply:
But the static analysis task found 25 warnings and this job didn't fail. So maybe something else is also needed.

Reply:
Try SdtReport task

Reply:
So the "Guardian Break" task found 12 warnings but I don't know why they are skipped.

Guardian is searching for results that meet the given criteria to break the build.
  Results Query Summary:
    Tool Filters (Include): apiscan, armory, bandit, binskim, codesign, credscan, csrf, detekt, eslint, flawfinder, FortifySca, fxcop, gosec:Error, infersharp:Error, moderncop, policheck, psscriptanalyzer, roslynanalyzers, prefast:Warning, semmle, spotbugs, tslint, terrascan, templateanalyzer
    Baselines: default
    Suppression Sets: default
    Policy: Microsoft
  Found no breaking results.
  Active results: 0
  Skipped results: 12
    Baselined results: 0
    Suppressed results: 0
    Results excluded by tool filters: 0
    Results below minimum severity: 12
    Results classified as Pass: 0

Reply:
Anyway, at least the analyzer is working now. We can merge this PR first and improve it later.